cmd/zoekt-sourcegraph-indexserver/debug.go

```diff
@@ -147,6 +154,98 @@ func debugListIndexed() *ffcli.Command {
 	}
 }
 
+func debugQueue() *ffcli.Command {
+	var connectionTimeout time.Duration
+	var printHeader bool
+
+	var serverAddress string
+
+	fs := flag.NewFlagSet("debug queue", flag.ExitOnError)
+	fs.DurationVar(&connectionTimeout, "timeout", 10*time.Second, "max timeout for establishing a connection to a zoekt-sourcegraph-indexserver instance")
+	fs.BoolVar(&printHeader, "header", false, "whether to print the headers for each column")
+
+	fs.StringVar(&serverAddress, "address", "http://localhost:6072", "the address of the zoekt-sourcegraph-indexserver instance to connect to")
```

Comment:
Maybe make the port a global const and use it here and for server?

Reply:
do we need localhost here or could we just ask for the port? If we just go with the port you don't have to distinguish between `serverAddress` and `fullAddress` and can just call it `addr`.

Reply:
I thought it would be nice to have the ability to use this command on remote zoekt instances (e.g. if you were port forwarding a pod). 

What about having separate `-port` and `-hostname` flags?